File: Java/src/main/java/com/gildedrose/SulfurasItem.java
package com.gildedrose;

public class SulfurasItem extends Item {

    public SulfurasItem(String name, int sellIn, int quality) {
        super(name, sellIn, quality);
    }

    @Override
    protected void updateQualityBeforeSellIn() {
        // do nothing
    }

    @Override
    protected void updateSellIn() {
        // do nothing
    }

    @Override
    protected void updateQualityAfterSellIn() {
        // do nothing
    }
}

File: Kotlin/src/main/kotlin/com/gildedrose/SulfurasItem.kt
package com.gildedrose

class SulfurasItem(override val name: String, override var sellIn: Int, override var quality: Int) : Item(name, sellIn, quality) {

    override fun updateQuality() {
    }

    override fun updateSellIn() {
    }

    override fun updateQualityWhenExpired() {
    }
}